test/spec/modules/liveIntentIdSystem_spec.js

```diff
@@ -190,7 +190,7 @@ describe('LiveIntentId', function() {
 
   it('should not return a decoded identifier when the unifiedId is not present in the value', function() {
     const result = liveIntentIdSubmodule.decode({ params: { fireEventDelay: 1, additionalData: 'data' } });
-    expect(result).to.be.eql({});
+    expect(result).to.be.eql({ 'lipb': {'params': { 'fireEventDelay': 1, 'additionalData': 'data' }} });
```

Comment:
It looks like the test is broken. The input should be (value, config). The test is only passing in config and it is handled as `value`. This inout should be `({}, { params: { fireEventDelay: 1, additionalData: 'data' } })`

Reply:
I adjusted it